statsmodels/genmod/generalized_linear_model.py

```diff
@@ -809,12 +823,12 @@ def _fit_gradient(self, start_params=None, method="newton",
                                     maxiter=maxiter, full_output=full_output,
                                     method=method, disp=disp, **kwargs)
 
-        self.mu = self.predict(rslt.params)
-        self.scale = self.estimate_scale(self.mu)
+        mu = self.predict(rslt.params)
+        scale = self.estimate_scale(mu)
```

Comment:
But it looks like irls and regularized still attach mu and scale to the model
I guess we need consistency across methods (even if it's currently not used).